shared_code/common.py

```diff
@@ -41,3 +43,79 @@ def abe_to_eas_fields_query(query_string):
         query_string = re.sub(r"\b%s\b" % old,new,query_string)
 
     return query_string
+
+def avs_to_eas_fields_query(query_string):
```

Comment:
i think we we will also want to include `street_number_sfx`, and a mapping for `address_number_suffix`: `street_number_sfx as address_number_suffix`

Reply:
[7ca87cd](https://github.com/SFDigitalServices/address-microservice-fn-py/pull/4/commits/18dfea7f6a24ed7a889f3f725065d17f327c096d)